frontend-public/src/components/dashboard/UserDashboard.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import PropTypes from "prop-types";
+import { Row, Col } from "antd";
+import { Link } from "react-router-dom";
+
+import { getUserInfo } from "@/selectors/authenticationSelectors";
+import { getUserMineInfo } from "@/selectors/userMineInfoSelector";
+import { fetchUserMineInfo } from "@/actionCreators/userDashboardActionCreator";
+import NullScreen from "@/components/common/NullScreen";
+import CustomPropTypes from "@/customPropTypes";
+import QuestionSidebar from "@/components/common/QuestionsSidebar";
+import Loading from "@/components/common/Loading";
+import * as routes from "@/constants/routes";
+
+const propTypes = {
+  userInfo: PropTypes.objectOf(PropTypes.string).isRequired,
```

Comment:
is objectOf needed here or can you just put PropTypes.string.isRequired?

Reply:
I think it is because it is an object that has many strings in it. It's not just one string.